src/EditorFeatures/Core/Implementation/InlineRename/IInlineRenameUndoManager.cs

```diff
@@ -20,7 +19,7 @@ internal interface IInlineRenameUndoManager : IWorkspaceService
     {
         void CreateInitialState(string replacementText, ITextSelection selection, SnapshotSpan startingSpan);
 
-        void CreateStartRenameUndoTransaction(Workspace workspace, ITextBuffer subjectBuffer, InlineRenameSession inlineRenameSession);
+        void CreateStartRenameUndoTransaction(Workspace workspace, ITextBuffer subjectBuffer, IInlineRenameSession inlineRenameSession);
```

Comment:
I can file a follow-up issue for this. I'm trying to avoid changes except those required during the work. I moved `IInlineRenameUndoManager` before moving `InlineRenameSession` which led to this change; I could not restore this to its original form or leave it like this.